src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -343,6 +344,32 @@ private void MergeTransactionsOnce()
             }
         }
 
+        private void CommitAndNotifyOnSlowWrite(DocumentsTransaction tx)
+        {
+            var sp = Stopwatch.StartNew();
+            tx.InnerTransaction.LowLevelTransaction.RetrieveCommitStats(out var stats);
+            tx.Commit();
+            sp.Stop();
+
+            if(stats.NumberOf4KbsWrittenToDisk == 0)
+                return;
+
+            var writtenDataInMb = stats.NumberOf4KbsWrittenToDisk / (double)256;
+            var seconds = sp.Elapsed.TotalSeconds;
+            var speed = writtenDataInMb / seconds;
+            // everything under 150ms will note raise an alert
+            if (seconds > 0.15 && (speed < 1 || seconds > 1))
```

Comment:
That's too low, and likely to cause a lot of spurious errors. 
May need to either increase the time or only warn if there have been more than one of these.

Reply:
This is why I think that anything below 150ms should _not_ generate a warning. Only if we occupy the disk for enough time to unstable the cluster (half of the election timeout).
So what numbers do you suggest ?